bump_rpm.sh

```diff
@@ -90,6 +90,16 @@ if [[ $CURRENT_VERSION != $NEW_VERSION ]] ; then
 		elif [[ -n $TEMPLATE ]] ; then
 			echo "* Updating requirements"
 			gem2rpm -t $ROOT/gem2rpm/$TEMPLATE.spec.erb *.gem | $SCRIPT_DIR/update-requirements specfile - $SPEC_FILE
+			if [[ $TEMPLATE == foreman_plugin ]]; then
+				UNPACKED_GEM_DIR=$(mktemp -d)
+				gem unpack --target "$UNPACKED_GEM_DIR" *.gem
+				PLUGIN_LIB="${UNPACKED_GEM_DIR}/${GEM_NAME}-${NEW_VERSION}/lib"
+				REQUIRES_FOREMAN=$(grep --recursive --no-filename requires_foreman $PLUGIN_LIB |sed -E 's/[^0-9.]//g')
```

Comment:
I always forget - is `.` something you need to escape in sed or don't?

Reply:
Yes.
Inside a character group (`[…]`) no, outside yes.
What you *need* to escape inside a character group is a `-` as it otherwise denotes a range (as `0-9` above) *unless* it's the last character of the group…